kitsune/wiki/forms.py

```diff
@@ -229,6 +229,8 @@ def save(self, parent_doc, **kwargs):
         if parent_doc:
             # Products are not set on translations.
             doc.products.remove(*[p for p in doc.products.all()])
+            # A child always inherits parent topics.
+            doc.topics.add(*[t for t in parent_doc.topics.all()])
```

Comment:
Why are you adding this in the DocumentForm?

Reply:
Honestly I put it there because the removal of `Products` removal code was there, which seemed to be analogous. I can try to add it to the view if you prefer, but I would probably need to modify the model's `save()` and some other bits (and probably some tests) so I can get an `.id` assigned to the `doc_form` so I can then assign it values from this relationship.  
I can look at that if you want, just let me know.